### PR TITLE
Refactor Documentation

### PR DESCRIPTION
**Pull request by matthewrsj**
#37 #39 #40 #45

### CODE REVIEW

**Comment by matthewrsj** [comment]
@osuosl/devs

File: docs/source/usage.rst [hunk 1]
```diff
@@ -3,23 +3,44 @@
 pymesync - Communicate with a TimeSync API
 ==========================================
 
+.. contents::
+
 This module provides an interface to communicate with an implementation of the
 `OSU Open Source Lab`_'s `TimeSync`_ API. An example implementation in Node.js
```
Comment (ElijahCaine) [inline docs/source/usage.rst]: ``An implementation of the TimeSync API, built in Node.js, can be found `on Github`_.``

Comment (ElijahCaine) [inline docs/source/usage.rst]: ```TimeSync API`_``

File: docs/source/usage.rst [hunk 2]
```diff
@@ -3,23 +3,44 @@
...
 
-This module allows users to send times to TimeSync (**create_time()**), get a time
-or a list of times from TimeSync (**get_times()**), and get a project or list of
-projects from TimeSync (**get_projects()**).
+This module allows users to
+
+* Send times, projects, and activities to TimeSync (**create_time()**,
+  **create_project()**, **create_activity()**),
+* Update times, projects, and activities (**update_time()**,
+  **create_project()**, **update_activity()**)
```
Comment (ElijahCaine) [inline docs/source/usage.rst]: shouldn't this be ``**update_project()**``?

File: docs/source/usage.rst [hunk 3]
```diff
@@ -98,44 +135,106 @@ TimeSync.\ **create_time(parameter_dict)**
...
+    filtered by parameters passed to ``kwargs``. Returns a list of python
+    dictionaries containing the time information returned by TimeSync or an
+    error message if unsuccessful.
 
     ``kwargs`` contains the optional query parameters described in the
     `TimeSync documentation`_. If ``kwargs`` is empty, ``get_times()`` will
     return all times in the database. The syntax for each argument is
-    ``query=["parameter"]`` except for the ``id`` parameter which is
-    ``id=<integer-id>``.
+    ``query=["parameter1", "parameter2"]`` except for the ``id`` parameter which
```
Comment (ElijahCaine) [inline docs/source/usage.rst]: There is no id parameter. There *is* a **uuid** parameter.

File: docs/source/usage.rst [hunk 4]
```diff
@@ -98,44 +135,106 @@ TimeSync.\ **create_time(parameter_dict)**
...
     * ``end`` - filter time request by end date
+
+      - example: ``end=["2015-07-23"]``
+
     * ``revisions`` - either ``["true"]`` or ``["false"]`` to include revisions
       of times
+
+      - example: ``revisions=["true"]``
+
     * ``id`` - get specific time entry by time id
```
Comment (ElijahCaine) [inline docs/source/usage.rst]: Again, uuid not id.

File: docs/source/usage.rst [hunk 5]
```diff
@@ -98,44 +135,106 @@ TimeSync.\ **create_time(parameter_dict)**
...
 
+      - example: ``id=134``
+
     .. warning::
 
       If the ``id`` parameter is passed all other parameters will be ignored.
       For example, ``ts.get_times(id=12, user=["bob"])`` is equivalent to
```
Comment (ElijahCaine) [inline docs/source/usage.rst]: id != uuid

File: docs/source/usage.rst [hunk 6]
```diff
@@ -269,7 +454,7 @@ Example usage:
     ...    "date_worked": "2014-04-17",
     ...}
     >>> ts.create_time(params)
-    {u'object': {u'activities': [u'documenting'], u'date_worked': u'2014-04-17', u'notes': u'Worked on docs', u'project': u'ganeti-web-manager', u'user': u'username', u'duration': 12, u'issue_uri': u'https://github.com/', u'id': 1}, u'auth': {u'username': u'username', u'password': u'userpass', u'type': u'password'}}
+    [{u'object': {u'activities': [u'documenting'], u'date_worked': u'2014-04-17', u'notes': u'Worked on docs', u'project': u'ganeti-web-manager', u'user': u'username', u'duration': 12, u'issue_uri': u'https://github.com/', u'id': 1}, u'auth': {u'username': u'username', u'password': u'userpass', u'type': u'password'}}]
```
Comment (ElijahCaine) [inline docs/source/usage.rst]: s/id/uuid/

**Comment by LyonesGamer** [comment]
+1 Looks good to me.

**Comment by ElijahCaine** [comment]
+1 lgtm

**Pull request merged**